cpp/src/arrow/compute/exec/plan_test.cc

```diff
@@ -840,6 +841,7 @@ TEST(ExecPlanExecution, ScalarSourceScalarAggSink) {
       }))));
 }
 
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge issue here

Reply:
I'm working on this again this morning — I think I'm close to constructing the proper `arrow_dplyr_query` object

Reply:
Weird, this wasn't showing up in vscode, but was definitely there — should be fixed now regardless.